interpreter/cling/include/cling/Interpreter/RuntimePrintValue.h

```diff
@@ -0,0 +1,162 @@
+//--------------------------------------------------------------------*- C++ -*-
+// CLING - the C++ LLVM-based InterpreterG :)
+// author:  Boris Perovic <boris.perovic@cern.ch>
+//
+// This file is dual-licensed: you can choose to license it under the University
+// of Illinois Open Source License or the GNU Lesser General Public License. See
+// LICENSE.TXT for details.
+//------------------------------------------------------------------------------
+#ifndef CLING_RUNTIME_PRINT_VALUE_H
+#define CLING_RUNTIME_PRINT_VALUE_H
+
+#include <string>
+#include "Value.h"
+
+#ifndef _Bool
+#define _Bool bool
```

Comment:
Is this really still needed? Can you show me how it's needed, i.e. what happens without it?